lib/active_record_shards/connection_specification.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-class ActiveRecord::Base
+ActiveRecord::Base.class_eval do
```

Comment:
if Activerecord::Base is not defined it will blow up instead of defining it

On Sat, Aug 20, 2016 at 8:45 AM, Benjamin Quorning <notifications@github.com
> wrote:

> In lib/active_record_shards/connection_specification.rb
> <https://github.com/zendesk/active_record_shards/pull/81#discussion_r75580606>
> :
>
> > @@ -1,5 +1,5 @@
> >  # frozen_string_literal: true
> > -class ActiveRecord::Base
> > +ActiveRecord::Base.class_eval do
>
> no accidental override
>
> Please clarify.
>
> silences rubocop warning about class definition style (compact vs nested)
>
> This would also be accomplished with:
>
> module ActiveRecord
>   class Base
>     # evil stuff
>   endend
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/active_record_shards/pull/81/files/ea7e0956e063e2513f944f774512905e3ff53097#r75580606>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyfx5QIWQ6GL00pIsYO4tR1eZNcFks5qhyEmgaJpZM4Jo-sL>
> .
>